terraform/main.tf

```diff
@@ -60,7 +61,6 @@ resource "null_resource" "update_ydns" {
     type        = "ssh"
     user        = "root"
     host        = digitalocean_droplet.wg.ipv4_address
-    private_key = file("~/.ssh/id_rsa")
```

Comment:
I stuck this in because
* This is the default anyways
* Not including this will allow terraform to use your ssh-agent to use passphrase-protected keys